### DISCUSSION TITLE
Combine search fields

### DISCUSSION THREAD

**Discussion by ahmeddedahi** [refinedev/refine#5878] [General]
Hi there,

I hope this message finds you well. I've been working on enhancing our search functionality in our Supabase and Next.js application, specifically focusing on combining search across multiple fields simultaneously.

Currently, we have data structured in a way where items are represented in different languages, such as English, French, and Arabic. Each item has corresponding fields for each language, like name_en, name_fr, and name_ar.

Now, I'm aiming to implement a search feature that allows users to search by any of these fields seamlessly. For instance, if a user types "Kitchen," they should get results even if not  the term matches with name_en or name_fr. Similarly, if they type "Appareils," they should also get relevant results.

const { tableProps , searchFormProps   } = useTable({

        syncWithLocation: true,
        resource: "category",

        initialFilter: [
            {
                field: "name->>name_en",
                operator: "ne",
                value: uuidv4(),

            },
            {
                field: "parent_id",
                operator: "null",
                value: "",
            },
        ],
    onSearch: (values:any) =>
        [
            {
                field: "name->>name_fr",
                operator: "contains",
                value: values.name,

            },

          {
              field: "name->>name_ar",
              operator: "contains",
              value: values.name,

          },
          {
              field: "name->>name_en",
            operator: "contains",
            value: values.name,

          },

      ],

  });

**Comment by BatuhanW** [comment]
Hey @ahmeddedahi you can use `or` filter to chain multiple filters.
